db/controllers/data.controller.js

```diff
@@ -0,0 +1,11 @@
+
+const pool = require('../connection.js')
+const queries = require('../queries/queries.js')
+
+exports.topicsRoute = (req, res) => {
+  pool.query(queries.getTopics, (err, results) => {
+    if (err) throw err;
+    res.status(200).json(results.rows)
+  })
+  
```

Comment:
A catch block would be good here rather than throwing your error so that it will use the middleware 🤸